app/controllers/samples_controller.rb

```diff
@@ -969,14 +969,24 @@ def results_folder
   end
 
   # GET /samples/:id/stage_results
+  # GET /samples/:id/stage_results.json
   def stage_results
     if current_user.allowed_feature_list.include?("pipeline_viz")
-      @results = {}
-      @sample.first_pipeline_run.pipeline_run_stages.each do |stage|
+      pipeline_run = @sample.first_pipeline_run
+      pipeline_version = pipeline_run.report_info_params[:pipeline_version]
```

Comment:
Why not just use `pipeline_run.pipeline_version` ?

Reply:
Updated — didn't realize pipeline_version could be accessed straight from pipeline_run!